src/main/java/com/uspray/uspray/DTO/auth/request/MemberDeleteDto.java

```diff
@@ -0,0 +1,16 @@
+package com.uspray.uspray.DTO.auth.request;
+
+import com.uspray.uspray.Enums.WithdrawReason;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import lombok.Data;
+
+@Data
+public class MemberDeleteDto {
+
+    @Schema(example = "[DONT_NEED, USE_DIFFICULT]", type = "array")
```

Comment:
```suggestion
    @Schema(example = ["DONT_NEED", "USE_DIFFICULT"], type = "array")
```
이렇게 바꿔야 swagger 상에서 바로 적용 가능하네요!